_includes/sidebar-releases.json

```diff
@@ -1,70 +1,37 @@
+{% assign today = "today" | date: "%Y-%m-%d" %}
+{% assign v_prod = site.data.versions | where_exp: "v_prod", "v_prod.asst_supp_exp_date > today" | where_exp: "v_prod", "v_prod.release_date < today" | sort: "release_date" | reverse | map: "major_version" %}
+{% assign v_test = site.data.versions | where_exp: "v_test", "v_test.release_date > today or v_test.release_date == 'N/A'" | sort: "release_date" | last | map: "major_version" %}
 {
   "title": "CockroachDB Releases",
   "items": [
     {
-      "title": "Production Releases",
-      "items": [
-        {
-          "title": "Latest v21.2",
-          "urls": [
-            "/releases/v21.2.5.html"
-          ]
-        },
-        {
-          "title": "Latest v21.1",
-          "urls": [
-            "/releases/v21.1.13.html"
-          ]
-        },
-        {
-          "title": "Latest v20.2",
-          "urls": [
-            "/releases/v20.2.19.html"
-          ]
-        },
-        {
-          "title": "All Production Releases",
-          "urls": [
-            "/releases/index.html#production-releases",
-            "/releases/v21.2.5.html",
-            "/releases/v21.2.4.html",
-            "/releases/v21.2.3.html",
-            "/releases/v21.2.2.html",
-            "/releases/v21.2.1.html",
-            "/releases/v21.1.14.html",
-            "/releases/v21.1.13.html",
-            "/releases/v21.1.12.html",
-            "/releases/v21.1.11.html",
-            "/releases/v21.1.10.html",
-            "/releases/v21.1.9.html",
-            "/releases/v21.1.8.html",
-            "/releases/v21.1.7.html",
-            "/releases/v21.1.6.html",
-            "/releases/v21.1.5.html",
-            "/releases/v21.1.4.html",
-            "/releases/v21.1.3.html",
-            "/releases/v21.1.2.html",
-            "/releases/v21.1.1.html",
-            "/releases/v21.1.0.html",
-            "/releases/v20.2.19.html"
-          ]
-        }
+      "title": "All Releases",
+      "urls": [
+        "/releases/index.html"
       ]
     },
+    {% for x in v_prod %}
+      {% assign r_latest = site.data.releases | where_exp: "r_latest", "r_latest.major_version == x" | where: "withdrawn", "false" | sort: "release_date" | last | map: "version" %}
+      {
+        "title": "Latest {{ x }}",
+        "urls": [
+          "/releases/{{ r_latest }}.html"
+        ]
+      },
+    {% endfor %}
+    {% if v_test[0] %}
     {
       "title": "Testing Releases",
```

Comment:
In my earlier comment, I was imaging this would be `Latest Testing Release" and point just to the release notes from the latest testing release for the upcoming major version. If that's what this link will be, can you change the link text?